### PR TITLE
Stronghold features

### LINKED ISSUE #182: Stronghold system
A zonetype has been made, but all logic assumes it is one zone!

Design a system where many Stronghold-typed zones can be created, have exit anomalies, teleport cols for entrances and stronghold teleport items to enter (go to these cols).

Probably a table with zones and anomaly locations (might be more than 1 per zone!)

Maybe other yet unspecified behavior and logic.  Design team still thinking.

### LINKED ISSUE #183: Make "exits" to strongholds more obvious, or easy to use
Teleport Anomalies on Strongholds as exits are ok.. but might not be self-explanatory and players might get trapped if they don't think to use them.

Introduce new "exit" entitydefault that renders as either a rift, teleport, or something that we can add text and names clearly to make exits obvious

Extra step (but stretch) for: https://github.com/OpenPerpetuum/OP-Project/issues/182

### PR DESCRIPTION
**Pull request by MikeJeffers**
- Support DB backed table of Exit rifts (Closes: https://github.com/OpenPerpetuum/OP-Project/issues/182)
 - Add new special kind of rift (so exits will be obvious and have customizable text) (Closes: https://github.com/OpenPerpetuum/OP-Project/issues/183)
 - Added support for multiple StrongholdTeleportMobileTeleport items jumping to new/different zones

Requires: https://github.com/OpenPerpetuum/OPDB/pull/221

### CODE REVIEW

File: src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs [hunk 1]
```diff
@@ -993,6 +994,7 @@ private void RegisterEntities()
             RegisterUnit<TrainingKillSwitch>();
             RegisterUnit<Gate>();
             RegisterUnit<RandomRiftPortal>();
+            RegisterUnit<StrongholdExitRift>(); // TODO new exit rift
```
Comment (nref-dan) [inline src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs]: There's still a TODO here? We shouldn't be merging anything in that still has work remaining to complete.
If the TODO is complete, remove the comment :)

File: src/Perpetuum/Perpetuum.csproj [hunk 2]
```diff
@@ -1207,7 +1213,6 @@
     <Folder Include="Properties\" />
     <Folder Include="Services\ProductionEngine\Methods\" />
     <Folder Include="Services\Relay\RequestHandlers\" />
-    <Folder Include="Comparers\" />
```
Comment (nref-dan) [inline src/Perpetuum/Perpetuum.csproj:1210]: Where did this folder go? iirc it's what contained the `IEqualityComparer` for the timed player object

File: src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs (hunk 1 above)
Comment (MikeJeffers) [reply src/Perpetuum.Bootstrapper/PerpetuumBootstrapper.cs]: Yeah bad habit from early in the project where we weren't 100% on proper placement in the bootstrapper

File: src/Perpetuum/Perpetuum.csproj (hunk 2 above)
Comment (MikeJeffers) [reply src/Perpetuum/Perpetuum.csproj:1210]: Addressed in https://github.com/OpenPerpetuum/PerpetuumServer/pull/193

**Review by LoyalServant — APPROVED** [review]
I'm okay with it.
The remaining todo items really need their own PR and a lot of other work done beyond the scope of this.

**Pull request merged**